lib/library.rb

```diff
@@ -0,0 +1,46 @@
+require 'book'
+require 'date'
+
+class Library
+   attr_accessor :books   #TODO duedate person status
+
+   def initialize(library)
+    @books = []
+   end
+
+   def check_in(book)
+    add = {item: book, status: :okay, person: :non, duedate: " " }
+    @books << add
+   end
+
+   def lend(title)
+    case
+     when books.detect{|a| a[:title] == title}[:status] = :lended
```

Comment:
Style/CaseIndentation: Indent `when` as deep as `case`. (https://github.com/bbatsov/ruby-style-guide#indent-when-to-case)